spec/lib/patches/db/pg_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'pg'
+
+require './lib/patches/db/pg'
+
+require 'spec_helper'
+
+describe PG::Connection do
+  let(:conn) { described_class.open(dbname: 'test') }
```

Comment:
oh yeah ... this is a nasty problem ... maybe just instead of making a DB connections here we can pass it various strings and binds and see that it behaves in a sane way?